testing/sdk_install.py

```diff
@@ -133,9 +134,30 @@ def install(
     _installed_service_names.add(service_name)
 
 
+def run_janitor(service_name, role, service_account, znode):
+    deslashed_service_name = sdk_utils.get_deslashed_service_name(service_name)
+
+    if role is None:
+        role = deslashed_service_name + '-role'
+    if service_account is None:
+        service_account = service_name + '-principal'
+    if znode is None:
+        znode = 'dcos-service-' + deslashed_service_name
+
+    cmd = ('docker run mesosphere/janitor /janitor.py '
```

Comment:
For complex commands like this (with a lot of `format` specifiers), I generally prefer to use a list and then generate the command using `" ".join(cmd_list)`. For example:
```python
    auth_token = sdk_cmd.run_cli('config show core.dcos_acs_token', print_output=False).strip()

    cmd_list = ["docker", "run", "mesosphere/janitor", "/janitor.py",
                "-r", role,
                "-p", service_account,
                "-z", znode,
                "--auth_token={}".format(auth_token)]
    cmd = " ".join(cmd_list)

    shakedown.run_command_on_master(cmd)
```